contracts/schemes/ArcScheme.sol

```diff
@@ -24,35 +32,62 @@ contract ArcScheme is Initializable {
     /**
      * @dev _initializeGovernance
      * @param _avatar the scheme avatar
-     * @param _votingMachine the scheme voting machine
-     * @param _voteParamsHash the scheme vote params
      * @param _votingParams genesisProtocol parameters - valid only if _voteParamsHash is zero
-     * @param _voteOnBehalf genesisProtocol parameter - valid only if _voteParamsHash is zero
+     * @param _voteOnBehalf  parameter
+     * @param _daoFactory  DAOFactory instance to instance a votingMachine.
+     * @param _stakingToken (for GenesisProtocol)
+     * @param _organization organization
+     * @param _callbacks should fulfill voting callbacks interface
+     * @param _authorizedToPropose only this address allow to propose (unless it is zero)
+     * @param _packageVersion packageVersion to instance the votingMachine from.
+     * @param _votingMachineName the votingMachine contract name.
      */
     function _initializeGovernance(
         Avatar _avatar,
-        IntVoteInterface _votingMachine,
-        bytes32 _voteParamsHash,
         uint256[11] memory _votingParams,
-        address _voteOnBehalf
+        address _voteOnBehalf,
+        DAOFactory _daoFactory,
+        address _stakingToken,
+        address _organization,
```

Comment:
Why is there both `_organization ` and `_avatar`?
Shouldn't we just have `_avatar `?